### PR TITLE
Import Tippy 6 directly

### PR DESCRIPTION
**Pull request by nstringham**
This PR makes tippy an npm dependency instead of using the CDN. this allows us to tree shake and also means that the browser doesn't have to open an extra connection. it also means that dependabot can update tippy now.

This PR also updates from tippy 5 to tippy 6

### CODE REVIEW

File: public/resources/ts/common-defer.ts [hunk 1]
```diff
@@ -143,9 +142,7 @@ function setCheckedTheme(theme: string) {
 
 setCheckedTheme(localStorage.getItem("currentTheme") ?? "default");
 
-tippy("*[data-tippy-content]", {
-  boundary: "window",
```
Comment (dukio) [inline public/resources/ts/common-defer.ts:147]: any clue why this was using `boundary: window` instead of the default `scrollParent`?

Comment (nstringham) [reply public/resources/ts/common-defer.ts:147]: I remember it was me that added it and it fixed a bug that we had been trying to fix for a long time but I can't remember the details

Comment (nstringham) [reply public/resources/ts/common-defer.ts:147]: here is where I did it https://github.com/SkyCryptWebsite/SkyCrypt/commit/eac7a231b62d5c9e9b080b2d444eb2160a8fe531

Comment (nstringham) [reply public/resources/ts/common-defer.ts:147]: it seams to be fixing #204

Comment (nstringham) [reply public/resources/ts/common-defer.ts:147]: after some testing this PR does not appear to reintroduce #204

**Review by MartinNemi03 — APPROVED** [review]

**Pull request merged**